### DISCUSSION TITLE
Linear transformation and units for ADC sensors

### DISCUSSION THREAD

**Discussion by msetina** [arendst/Tasmota#19200] [Q&A]
I can not find parameters that could be used for ADC sensors to apply linear transformation (K, N from Kx+N) or units for the measured value.
If I had a current clamp on the other side of ADC Tasmota could show current in A instead of ADC conversion value.

**Comment by sfromis** [comment]
You can get a linear transformation by configuring the pin as `ADC Range` and using the [AdcParam](https://tasmota.github.io/docs/ADC/#commands) command linear range remapping parameters.

If you instead want to use the existing Tasmota feature for a CT clamp, you can instead configure the pin as `ADC CT Power` and the CT POWER parameter adjustments of the AdcParam command.
